src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -86,6 +90,20 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, reimbursementAcc
     const activeRoute = useNavigationState(getTopmostWorkspacesCentralPaneName);
     const {translate} = useLocalize();
     const {canUseAccountingIntegrations} = usePermissions();
+    const {isOffline} = useNetwork();
+
+    const prevPendingFields = usePrevious(policy?.pendingFields);
+    const policyFeatureStates = useMemo(
+        () => ({
+            [CONST.POLICY.MORE_FEATURES.ARE_DISTANCE_RATES_ENABLED]: policy?.areDistanceRatesEnabled,
+            [CONST.POLICY.MORE_FEATURES.ARE_WORKFLOWS_ENABLED]: policy?.areWorkflowsEnabled,
+            [CONST.POLICY.MORE_FEATURES.ARE_CATEGORIES_ENABLED]: policy?.areCategoriesEnabled,
+            [CONST.POLICY.MORE_FEATURES.ARE_TAGS_ENABLED]: policy?.areTagsEnabled,
+            [CONST.POLICY.MORE_FEATURES.ARE_TAXES_ENABLED]: policy?.tax?.trackingEnabled,
+            [CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED]: policy?.areConnectionsEnabled,
+        }),
+        [policy],
+    ) as PolicyFeatureStates;
```

Comment:
This cast `as PolicyFeatureStates` is wrong, we shouldn't cast except for very very very exceptional cases. If you remove the cast, we can clearly see that the type here is wrong:

<img width="1010" alt="image" src="https://github.com/user-attachments/assets/a6032d6e-db0c-4ecf-b31e-985d0f2d9475">

You are saving possibly `undefined` values into a type that is supposed to have `boolean` (not `boolean | undefined`)